static/templates/subscription_setting_icon.handlebars

```diff
@@ -2,6 +2,8 @@
     <div class="flex">
         {{#if invite_only}}
         <i class="fa fa-lock" aria-hidden="true"></i>
```

Comment:
I don't know why we didn't use `fa-lg` here and added a custom size. I for the globe icon used fa-lg and it seems nice. Is there a particular reason for not doing that?
Look for the fa-lg just below this comment.

Reply:
Probably historical -- this HTML may predate our upgrading to whatever version of font awesome added `fa-lg`.  Better to use that if it looks good.